app/models/organization.rb

```diff
@@ -65,4 +67,13 @@ def active_loans
     loans.where(status_value: Loan::STATUS_ACTIVE_VALUE)
   end
 
+  private
+
+  def primary_contact_is_member
+    return if primary_contact.blank?
+    return if person_ids.include?(primary_contact_id)
+
+    errors.add(:primary_contact, I18n.t('organization.invalid_primary_contact'))
```

Comment:
Please use the proper Rails I18n conventions for this.

I believe it would be `activerecord.errors.models.organization.attributes.primary_contact.invalid`.

Then you just do `errors.add(:primary_contact, :invalid)`.